### PR TITLE
Benkuhn/add to more pages

### PR DESCRIPTION
**Pull request by BenJKuhn**
This adds some robustness changes that address a race in loading the document. It also adds the links to the pull requests list page.

### CODE REVIEW

File: codeflow.js [hunk 1]
```diff
@@ -1,18 +1,80 @@
-var discussionHeaders = document.getElementsByClassName("gh-header-number");
-var discussionHeader = discussionHeaders[0];
-var prlink = document.location;
-
-var codeflowElement = document.createElement("span");
+(function(){
```
Comment (henrik-me) [inline codeflow.js:1]: generally looks great. I made some changes yesterday and today to optimize the link for various scenarios such as # to specific changes and files tab/files change links. Thus this needs a minor update to ensure those changes stick. #Closed

File: manifest.json [hunk 2]
```diff
@@ -12,8 +12,9 @@
     
       "content_scripts": [
         {
-          "matches": ["*://*.github.com/*/pull/*"],
-          "js": ["codeflow.js"]
+          "matches": [ "*://*.github.com/*/pull/*", "*://*.github.com/*/pulls*"],
+          "js": [ "codeflow.js" ],
+          "run_at": "document_idle"
```
Comment (henrik-me) [inline manifest.json:17]: >run_at": "document_idle" [](start = 11, length = 24)

When on the pr page it's possible to switch between the conversation tab and the files changed tab. We might have to do a bit more to ensure the changes are applied in that case as well. Can be applied on top of your change. #Pending

File: codeflow.js (hunk 1 above)
Comment (BenJKuhn) [reply codeflow.js:1]: I've re-applied the changes that you made into the new code I wrote. #Closed

File: codeflow.js [hunk 3]
```diff
@@ -1,41 +1,94 @@
...
+    if (applyLinks() == false) 
+    {
+        setTimeout(tryUntilSuccess(), 500);
+    }
+}
+
+tryUntilSuccess();
+
+// window.onload = tryUntilSuccess;
+// document.addEventListener("pageshow", tryUntilSuccess)
```
Comment (henrik-me) [inline codeflow.js]: remove?

**Review by henrik-me — APPROVED** [review]
:shipit:

**Pull request merged**